README.md

```diff
@@ -77,11 +77,13 @@ A list of available commands is accessible by running `codeclimate` or
 $ codeclimate help
 
 Available commands:
-    analyze [-f format] [-e engine(:channel)] [--dev] [path]
+    analyze [-f format] [-e engine[:channel]] [path]
```

Comment:
I think we'd like to keep the `--dev` flag documented here, please.

Reply:
codeclimate prepare